### PR TITLE
minor fixes in tax_smoothing_3.rst

### PR DESCRIPTION
**Pull request by AnjuJoon**

### CODE REVIEW

**Comment by AnjuJoon** [comment]
@natashawatkins How does this look to you?  We can resolve the merge conflicts at this end.

File: source/rst/tax_smoothing_3.rst [hunk 1]
```diff
@@ -50,13 +59,13 @@ where :math:`w_{t+1} \sim {\cal N}(0,I)`. The variables
 while :math:`b_{t-1,t}` is an endogenous state variable inherited from
 the past at time :math:`t` and :math:`p^t_{t+1}` is an exogenous state
 variable at time :math:`t`. This is the same set-up as used `in this
-notebook <http://nbviewer.jupyter.org/github/QuantEcon/TaxSmoothing/blob/master/Tax_Smoothing_1.ipynb>`__.
+lecture <https://lectures.quantecon.org/py/tax_smoothing_1.html>`__
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_3.rst]: refer to other lectures using `:doc:`` `

File: source/rst/tax_smoothing_3.rst [hunk 2]
```diff
@@ -50,13 +59,13 @@ where :math:`w_{t+1} \sim {\cal N}(0,I)`. The variables
...
 We will consider a situation in which the government faces “roll-over
 risk”. Specifically, we shut down the government’s ability to borrow in
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_3.rst]: new line

File: source/rst/tax_smoothing_3.rst [hunk 3]
```diff
@@ -69,21 +78,21 @@ in Markov state 1 and
 
 \ in Markov state 2. Consequently, in the second Markov state the
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_3.rst]: new line

File: source/rst/tax_smoothing_3.rst [hunk 4]
```diff
@@ -122,139 +131,141 @@ The transition matrix for this formulation is:
...
-.. code:: ipython3
+.. code-block:: python3
 
     # Model parameters 
     beta, Gbar, rho, sigma = 0.95, 5, 0.8, 1
     
     # Basic model matrices
-    A22 = np.array([[1,0],[Gbar, rho],])
+    A22 = np.array([[1, 0], [Gbar, rho], ])
     C2 = np.array([[0], [sigma]])
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_3.rst]: unicode characters

File: source/rst/tax_smoothing_3.rst [hunk 5]
```diff
@@ -122,139 +131,141 @@ The transition matrix for this formulation is:
...
-    Q = np.dot(M.T,M)
-    W = np.dot(M.T,S)
+    Q = np.dot(M.T, M)
+    W = np.dot(M.T, S)
     
     # Create namedtuple to keep the R,Q,A,B,C,W matrices for each state of the world
     world = namedtuple('world', ['A', 'B', 'C', 'R', 'Q', 'W'])
     
-    Pi = np.array([[0.95,0,0.05,0],[0.95,0,0.05,0],[0,0.9,0,0.1],[0,0.9,0,0.1]])
+    Pi = np.array([[0.95, 0, 0.05, 0], [0.95, 0, 0.05, 0], [0, 0.9, 0, 0.1], [0, 0.9, 0, 0.1]])
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_3.rst]: align as matrix

File: source/rst/tax_smoothing_3.rst [hunk 6]
```diff
@@ -122,139 +131,141 @@ The transition matrix for this formulation is:
...
-    tax = np.zeros([T,1])
+    tax = np.zeros([T, 1])
     for i in range(T):
-        tax[i,:] = S.dot(x[:,i]) + M.dot(u[:,i])
+        tax[i, :] = S.dot(x[:, i]) + M.dot(u[:, i])
     
     #Plot of debt issuance and taxation
-    plt.figure(figsize=(16,4))
+    plt.figure(figsize=(16, 4))
     plt.subplot(121)
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_3.rst]: don't need these two lines

File: source/rst/tax_smoothing_3.rst [hunk 7]
```diff
@@ -122,139 +131,141 @@ The transition matrix for this formulation is:
...
     T = 300
-    x,u,w,state = MJLQBarro.compute_sequence(x0,ts_length=T)
+    x, u, w, state = MJLQBarro.compute_sequence(x0, ts_length=T)
     
     # Calculate taxation each period from the budget constraint and the Markov state
-    tax = np.zeros([T,1])
+    tax = np.zeros([T, 1])
     for i in range(T):
-        tax[i,:] = S.dot(x[:,i]) + M.dot(u[:,i])
+        tax[i, :] = S.dot(x[:, i]) + M.dot(u[:, i])
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_3.rst]: replace `.dot()`

**Comment by natashawatkins** [comment]
There's still a bit of cleaning up to do

**Pull request merged**